python/tvm/relay/frontend/pytorch.py

```diff
@@ -5880,18 +5892,26 @@ def outplace_inplace_ops(opnodes):
     for i, (node_name, op_node) in enumerate(opnodes):
         operator = op_node.kind()
         # Check if op is in-place    (avoid '__not__', etc.)
-        if operator[-1] == '_' and operator[-2:] != "__": 
+        if operator[-1] == '_' and operator[-2:] != "__":
             input_node = op_node.inputsAt(0)
             replace_map.append((i, input_node, op_node.outputsAt(0)))
 
     # Replace future uses of node with an in-place op applied to it with the output of the op
-    for node_idx, orig_node, replacement_node in replace_map:
-        relevant_ops = opnodes[node_idx+1:]
+    node_inputs_map = {}
```

Comment:
As this hash map can become heavy during runtime depending on the model size, make sure to delete it after usage at the end of this function. Garbage collector should take care of it at some point, but let's release it sooner so that we don't run into storage OOM cases later on during compilation. 

E.g. `del node_inputs_map`